lib/index.js

```diff
@@ -1,15 +1,12 @@
-
```

Comment:
Please don't change our code styles/formatting/whitespaces, especially when there are code changes in the PR. You can send these as a seperate PR if you feel strongly about the changes.